scripts/jointLklDM.C

```diff
@@ -722,15 +723,12 @@ void jointLklDM(TString configFileName="$GLIKESYS/rcfiles/jointLklDM.rc",Int_t s
       // Save -2logLkl vs <sv> in file
       //////////////////////////////////////////////////////////////
       if (exportData)
-        {
           for (Int_t isv=0; isv<=svNPoints; isv++)
             vlkl2D[isv][imass] = grLklParabola[imass]->Eval(svScan[isv]) - grLklParabola[imass]->Eval(0);
-        }
 
     } // end of loop over DM masses
 
     if (exportData)
```

Comment:
Move this 2 [?] spaces to the left

Reply:
Done.

Reply:
As explained in the e-mail, for gLike it is irrelevant the y-axis offset of the likelihood function. This is a GloryDuck convention and I see no strong reason to change it. Also, it provides an elegant way to know the value of the likelihood function at zero, which is needed for the computation of the detection significance.